pandas/core/index.py

```diff
@@ -20,7 +20,30 @@
     ensure_index_from_sequences,
     get_objs_combined_axis,
 )
-from pandas.core.indexes.multi import _sparsify  # noqa:F401
```

Comment:
Is this actually imported from anywhere else? If not I think could actually be removed